tests/framework/clients/rancher/provisioning/cluster.go

```diff
@@ -0,0 +1,97 @@
+package provisioning
+
+import (
+	"context"
+
+	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	rkev1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewRKE2ClusterConfig(clusterName, namespace, cni, cloudCredentialSecretName, kubernetesVersion string, machinePools []apisV1.RKEMachinePool) *apisV1.Cluster {
+	typeMeta := metav1.TypeMeta{
+		Kind:       "Cluster",
+		APIVersion: "provisioning.cattle.io/v1",
+	}
+
+	//metav1.ObjectMeta
+	objectMeta := metav1.ObjectMeta{
+		Name:      clusterName,
+		Namespace: namespace,
+	}
+
+	etcd := &rkev1.ETCD{
+		DisableSnapshots:     false,
+		S3:                   nil,
```

Comment:
are all of these default values? if so we don't need to set them

Reply:
yes these are the defaults, I believe.